programs/gummyroll/src/lib.rs

```diff
@@ -146,8 +146,8 @@ pub mod gummyroll {
         root: Node,
         leaf: Node,
         index: u32,
-        changelog_db_uri: String,
-        metadata_db_uri: String,
+        _changelog_db_uri: String,
+        _metadata_db_uri: String,
```

Comment:
this might break indexing of instructions, since the generated instruction uses the variable names as struct fields.